tf-psa-crypto/drivers/builtin/include/mbedtls/gcm.h

```diff
@@ -285,8 +285,8 @@ int mbedtls_gcm_update_ad(mbedtls_gcm_context *ctx,
  * \note            The output buffer \p output can be the same as the input
  *                  buffer \p input. If \p output is greater than \p input, they
  *                  cannot overlap. Implementations which require
- *                  MBEDTLS_GCM_ALT to be enabled may not provide support for
- *                  overlapping buffers.
+ *                  MBEDTLS_GCM_ALT //no-check-names to be enabled may not
```

Comment:
Ah. There's a good reason we have `check_names.py` telling us about this.

I'm very sorry, I forgot that we've removed `MBEDTLS_GCM_ALT` in `development`. Would you be able to remove the note about `MBEDTLS_GCM_ALT` in this PR? (But not in the backports).